app/index.js

```diff
@@ -22,25 +23,38 @@ process.chdir(__dirname);
 // Setup the app
 var port = process.env.PORT || 3000;
 app.listen(port);
-app.use(bodyParser.urlencoded({
-	extended: true
-}));
+app.use(
+  bodyParser.urlencoded({
+    extended: true
+  })
+);
 app.use(bodyParser.json());
 
 let spaces = 4;
-if(process.env.NODE_ENV === 'production') {
+if (process.env.NODE_ENV === 'production') {
   spaces = 0;
 }
 app.set('json spaces', spaces);
 
 // Set the version
 app.set('version', require('../package.json').version);
 
+exec(`git rev-list -n1 ${app.get('version')}`, (err, stdout, stderr) => {
+  if (err) {
+    console.log(err);
+  } else {
+    console.log('stdout', stdout);
+    console.log('stderr', stderr);
```

Comment:
Let's not use console.log here, but rather the logging helper in `app/helpers/logger.js`